src/sections/Brand/Brand-components/layer5.js

```diff
@@ -121,121 +141,45 @@ const Layer5 = () => {
                     </Col>
                 </Row>
                 <Row>
-                    <Col xs={12} sm={6} lg={6} xl={6}>
-                        <h3 className="layerH3">
-                            <strong>Colors</strong>
-                        </h3>
+                    <Col>
+                        <h1 className="layerH3 in">
+                                Colors
+                        </h1>
                     </Col>
-                    <Col xs={12} sm={12} lg={12} xl={12}>
+                </Row>
+                <Row>
+                    <Col>
                         <p>
-                            Our community color palette consists of gray, green and colors
-                            representing each of our projects. The Layer5 logo should be white
-                            when using project colors as the background.
+                    Our community color palette consists of gray, green and colors
+                    representing each of our projects. The Layer5 logo should be white
+                    when using project colors as the background.
                         </p>
                     </Col>
                 </Row>
-                <Col xs={12} sm={12} lg={12} xl={12}>
-                    <h4 className="layerH3">Web</h4>
-                </Col>
                 <Row>
-                    <Col xs={12} sm={12} md={6} lg={6} xl={6}>
-                        <Row>
-                            <Col xs={12} sm={12} lg={12} xl={12}>
-                                <p className="layerH3">Color</p>
-                            </Col>
-                            <ColorBox R="R: 60" G="G: 73" B="B: 79" colorCode="#3c494f" />
-                            <ColorBox R="R: 0" G="G: 179" B="B: 159" colorCode="#00b39f" />
-                            <ColorBox R="R: 0" G="G: 211" B="B: 169" colorCode="#00d3a9" />
-                        </Row>
-                    </Col>
-                    <Col xs={12} sm={12} md={6} lg={6} xl={6}>
-                        <Row>
-                            <Col xs={12} sm={12} lg={12} xl={12}>
-                                <p className="layerH3">Grayscale</p>
-                            </Col>
-                            <ColorBox R="R: 60" G="G: 73" B="B: 79" colorCode="#3c494f" />
-                            <ColorBox R="R: 130" G="G: 140" B="B: 140" colorCode=" #828c8c" />
-                            <ColorBox R="R: 160" G="G: 170" B="B: 170" colorCode="#a0aaaa" />
-                        </Row>
-                    </Col>
+                    <ColorBox name="Keppel" R="0" G="179" B="159" colorCode="#00B39F" />
+                    <ColorBox name="Caribbean Green" R="0" G="211" B="169" colorCode="#00D3A9" />
+                    <ColorBox name="Saffron" R="235" G="192" B="23" colorCode="#EBC017" />
+                    <ColorBox name="Blond" R="255" G="243" B="197" dark colorCode="#FFF3C5" />
```

Comment:
If it's simple enough to make the font color darker in the Blond color box, that might improve its legibility.